test/ducks/apps/__snapshots__/index.spec.js.snap

```diff
@@ -181,56 +181,8 @@ Array [
     "type": "FETCH_APP",
   },
   Object {
-    "app": Object {
-      "categories": Array [
-        "cozy",
-      ],
-      "developer": Object {
-        "name": "Cozy",
-      },
-      "editor": "Cozy",
-      "icon": "<svg></svg>",
-      "installed": true,
-      "isInRegistry": true,
-      "label": 1,
-      "links": Object {
-        "icon": "/registry/collect/3.0.0/icon",
-      },
-      "name": "Collect",
-      "name_prefix": "Cozy",
-      "permissions": Object {
-        "mock": Object {
-          "type": "io.mock.doctype",
-        },
-        "mock2": Object {
-          "type": "io.mock.doctype2",
-        },
-      },
-      "slug": "collect",
-      "tags": Array [
-        "konnector",
-        "collect",
-        "bills",
-        "providers",
-        "files",
-      ],
-      "type": "webapp",
-      "uninstallable": true,
-      "version": "3.0.0",
-      "versions": Object {
-        "beta": Array [
-          "3.0.0",
-        ],
-        "dev": Array [
-          "3.0.0",
-        ],
-        "stable": Array [
-          "3.0.0",
-        ],
-      },
-    },
-    "lang": "en",
-    "type": "FETCH_APP_SUCCESS",
+    "error": [Error: Error while getting the application with slug: collect],
```

Comment:
I think there is an error inside the test 🤔

Reply:
cozy-stack-client had to be updated to work with cozy-client. I didn't notice that when updating cozy-client